### PR TITLE
Pin blosc to 1.14.3 for travis-27 build

### PR DESCRIPTION
**Pull request by TomAugspurger**
This reverts commit 52e186af3e0fed9415a2db9e0c8d7d75d1267a54.

xref https://github.com/pandas-dev/pandas/issues/22427 and https://github.com/pandas-dev/pandas/pull/22424

### CODE REVIEW

File: pandas/tests/extension/integer/test_integer.py [hunk 1]
```diff
@@ -770,7 +770,7 @@ def test_groupby_mean_included():
     df = pd.DataFrame({
         "A": ['a', 'b', 'b'],
         "B": [1, None, 3],
-        "C": IntegerArray([1, None, 3], dtype='Int64'),
```
Comment (jorisvandenbossche) [inline pandas/tests/extension/integer/test_integer.py]: I would do this in a separate PR

Comment (TomAugspurger) [reply pandas/tests/extension/integer/test_integer.py]: Keeping it here, now that we know that the doc PR isn't related.
